test-data/unit/pythoneval.test

```diff
@@ -1367,3 +1367,49 @@ x: Dict[str, List[int]]
 reveal_type(x['test'][0])
 [out]
 _testNewAnalyzerBasicTypeshed_newsemanal.py:4: error: Revealed type is 'builtins.int*'
+
+[case testDecoratorDecorator]
```

Comment:
The existing `pythoneval` tests typically test some tricky but important corner cases where mypy used to not work. We only try to add `pythoneval` tests when they are really necessary, because they are very slow. I would propose to just keep the above fix in this PR.

Reply:
Also, `decorator` is a third-party library; I don't think it makes sense for mypy to test the stubs for third-party libraries.